### PR TITLE
Use US-English color in visible text, preserving incoming links that …

### PR DESCRIPTION
**Pull request by svgeesus**
…use colour #374

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -843,26 +843,26 @@ <h2 id="4Concepts.Sourceimage">Images</h2>
     <section id="4Concepts.ColourSpaces">
       <h2>Colour spaces</h2>
```
Comment (annevk) [inline index.html:844]: Seems like this should be renamed. (Not the ID of course.)

File: index.html [hunk 2]
```diff
@@ -1021,14 +1021,14 @@ <h2>PNG image</h2>
...
-        <dt><dfn>Truecolour with alpha</dfn></dt>
+        <dt><dfn>Truecolor with alpha</dfn></dt>
         <dd>Each <a>pixel</a> consists of four <a>samples</a>: red, green, blue and <a>alpha</a>.</dd>
 
         <dt><dfn>Greyscale with alpha</dfn></dt>
         <dd>Each <a>pixel</a> consists of two <a>samples</a>: a <a>grey sample</a> and an <a>alpha</a> sample.</dd>
 
         <!-- Maintain a fragment named "3truecolour" to preserve incoming links to it -->
-        <dt><dfn id="3truecolour">Truecolour</dfn></dt>
+        <dt id="truecolor"><dfn id="3truecolour">Truecolor</dfn></dt>
```
Comment (annevk) [inline index.html:1031]: I would either not add an ID or give the `<dfn>` the improved ID.

File: index.html [hunk 3]
```diff
@@ -5618,17 +5618,17 @@ <h2>Interlacing</h2>
     <section id="12Filter-selection">
       <h2>Filter selection</h2>
 
-      <p>For images of <a>colour type</a> 3 (indexed-colour), filter type 0 (None) is usually the most effective. Colour images
-      with 256 or fewer colours should almost always be stored in <a>indexed-colour</a> format; <a>truecolour</a> format is likely to be
+      <p>For images of <a>color type</a> 3 (indexed-color), filter type 0 (None) is usually the most effective. Colour images
```
Comment (annevk) [inline index.html:5621]:
```suggestion
      <p>For images of <a>color type</a> 3 (indexed-color), filter type 0 (None) is usually the most effective. Color images
```
Maybe do another search that's case-insensitive.

**Review by annevk — COMMENTED** [review]
Thanks! Style guides FTW. (Not approving as I'm not a reviewer.)

File: index.html (hunk 1 above)
Comment (svgeesus) [reply index.html:844]: Good catch! I see that I had "match case" on in my search-and-replace, so I just did another pass changing Colour to Color as needed.

File: index.html (hunk 2 above)
Comment (svgeesus) [reply index.html:1031]: We want to preserve inbound link fragments where possible, to keep compatibility from the 2003 second edition. That is why this one has both; it enables ReSpec `<a>truecolor</a>` to work while also preserving the older id as well.

Comment (annevk) [reply index.html:1031]: ReSpec doesn't key off the `dfn`? Weird but okay.

**Review by svgeesus — COMMENTED** [review]
Thanks for those, good to have consistent usage

**Comment by svgeesus** [comment]
@ProgramMax does this look ok to you?

**Review by chrisn — APPROVED** [review]

**Pull request merged**